### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by fortuneonyeka**
In this project, I build some forms, both using nearly pure HTML and the helper methods from Rails. Note that this project doesn't intend to provide any styles to the created views, just the bare forms. The following controllers have been created:

 #new
 #create
 #edit
 #update

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>"/>
+    <label for="username">Username:</label>
+    <input id="username" name="user[username]" type="text" />
+    <label for="email">Email:</label>
+    <input id="email" name="user[email]" type="text" />
+    <label for="password">Password:</label>
+    <input id="password" name="user[password]" type="password" />
+    <br>
+    <input name="submit" type="submit" value="Submit" />
+    </form>
```
Comment (BrianSammit) [inline app/views/users/new.html.erb]: In line 39 instead of `action= "/users"` use the users_path helper `action="users_path"`

**Review by BrianSammit — CHANGES_REQUESTED** [review]
CHANGES REQUIRED :red_circle: :stop_sign: 
---------------
Hello @fortuneonyeka, great job you did in this project so far :clap: 
TO HIGHLIGHT: 

1. Your project is professional 
2. Your repo is professional
3. Well use of the MVP 

However, there are a few changes to make to complete this project :hammer_and_wrench: see them below :eyes: :point_down: 
---------------

- [x] Please fix the broken image in the readme file. 
- [x] Please fix the MIT link that is broken on your readme file.
- [x] Please check the steps in your readme file in section INSTALL before you migrate the database you need to create it `rails db:create`
- [x] Please remove the space between `:` and the `install` in this line of code `bundle exec rake webpacker: install` this space creates a conflict and does not run. Remember that not all people that going to see the code not about programming. 
- [x] Same as above `rails db: migrate` :point_up_2: 
- [x] I got this error when I try to create a new user. 
Screenshot from 2021-06-18 20-08-11

check the `user_controller ` instead of `@user = User.new[user_params]` use `@user = User.new(user_params)` the only thinks that change is the square brackets to parentheses. 

- [x] Please check my inline comments :point_down: 

------------
Please make these required changes :hammer_and_wrench: and send it again for another review :+1: :muscle: 
Reachme on SLACK if you have any questions. 
@ Brian Sammit Cruz Rodriguez 
HAPPY CODING!!!

tenor (1)

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️

Hello @fortuneonyeka,

Good job applying most of the previous TSE required fixes, however, you are still missing one small detail:

- [x] Your Readme MIT link is still broken

Please correct those issues and send another review request, good luck, and well done!

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hello, @fortuneonyeka,

You did a great job with this project, good luck with the next ones!

**Pull request merged**